### ISSUE TITLE
Conflict with PropertySpacingSniff and ConstantSpacingSniff

### ISSUE THREAD

**Issue by arxeiss** [slevomat/coding-standard#880]
If I set no spacing for constants without comments in `ConstantSpacingSniff`, because I have also `UselessConstantTypeHint`. But then set some spacing for properties, there is a conflict.

The reason is simple, if I use visibility for constants as well, PropertySpacingSniff catch also constants. Because there is public/private/protected keyword.

**Comment by kukulich** [comment]
@arxeiss please post code that can reproduce the error

**Comment by arxeiss** [comment]
Here is the example. However I realised there is another inconsistency.

The following code produce this errors. But why there is "Expected 1 to 1 blank lines **after** property, found 0" when all properties are called min/maxLinesCount**Before**With(out)Comment.

```
FILE: src/Example.php
----------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------------------------------------------------------------------------------------------------
  9 | ERROR | [x] Expected 1 to 1 blank lines after property, found 0.
    |       |     (SlevomatCodingStandard.Classes.PropertySpacing.IncorrectCountOfBlankLinesAfterProperty)
 10 | ERROR | [x] Expected 0 to 0 blank lines after constant, found 1.
    |       |     (SlevomatCodingStandard.Classes.ConstantSpacing.IncorrectCountOfBlankLinesAfterConstant)
```

```php
class Example
{
	public const FIRST_PAGE_FORMAT = '';
	public const PAGE_FORMAT = '?page=%d';

	/** @var int */
	protected $currentPage;

	/** @var int */
	protected $totalPages;
}
```

```xml
<rule ref="SlevomatCodingStandard.Classes.ConstantSpacing">
    <properties>
        <property name="minLinesCountBeforeWithComment" value="0"/>
        <property name="maxLinesCountBeforeWithComment" value="0"/>
        <property name="minLinesCountBeforeWithoutComment" value="0"/>
        <property name="maxLinesCountBeforeWithoutComment" value="0"/>
    </properties>
</rule>
<rule ref="SlevomatCodingStandard.Classes.PropertySpacing">
    <properties>
        <property name="minLinesCountBeforeWithComment" value="1"/>
        <property name="maxLinesCountBeforeWithComment" value="1"/>
        <property name="minLinesCountBeforeWithoutComment" value="1"/>
        <property name="maxLinesCountBeforeWithoutComment" value="1"/>
    </properties>
</rule>
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/bb9db1ac5f7407b991f62b5e59c6a97e98df8280

**Comment by marcoocram** [comment]
I don't know if is related but PropertySpacing doesn't detect php 7.4 typed properties as properties and mix it with constants, can't be?

Thanks!
